templates/guides/migration-guide-1.0.0.md

```diff
@@ -0,0 +1,79 @@
+---
+page_title: "Fastly Terraform 1.0.0 Migration Guide"
+subcategory: "Migration"
+---
+
+## Fastly Terraform 1.0.0 Migration Guide
```

Comment:
This context seems a bit unnecessary.  Also I was unsure whether this was about migrating from 1 or to 1.  Finally, it's unclear whether this migration guide continues to apply when we move beyond 1.0.0 (eg to 1.0.1).  Consider "Upgrading to version 1"